tests/console/validate_raid.pm

```diff
@@ -73,7 +74,7 @@ my (
 );
 # Prepare test data depending on specific architecture/product
 sub prepare_test_data {
-    if (is_ppc64le || is_ppc64) {
+    if (is_pvm) {
```

Comment:
I think you can reserve the previous condition. is_pvm check backend is 'spvm' or 'pvm_hmc' so that doesn't work .  And is_ppc64le will check ARCH=aarch64 works for all test. Create a new condition for test data if something different on powerVM or power 10 kvm.